.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,71 @@
+name: Build wheel
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [windows-latest, ubuntu-latest, macos-latest, macos-14]
+    env:
+      CIBW_SKIP: 'pp*'
+      CIBW_ARCHS: 'auto64'
+      CIBW_PROJECT_REQUIRES_PYTHON: '>=3.9'
+      CIBW_TEST_REQUIRES: 'pytest'
+      CIBW_TEST_COMMAND: 'pytest {project}/tmp/tests'
+
+    defaults:
+      run:
+        shell: bash -l {0}
+     
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+      - uses: actions/checkout@v4      
+        with:
+          submodules: true
+      - name: Set up Python version ${{ matrix.version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.12"
```

Comment:
Is it possible to use something like "latest" here?

Reply:
Changed to 3.x